Clinical-Genomics/scout
# Except KeyError in Load Command

Hi,
This PR is to:
* Get feedback
* Push some work so I don't loose it 
* ...or Delete the branch if there is no need 

The PR is a small part of an idea I had, to make some configuration parameters mandatory and raise an exception if they were not present. We had a small discussion on the matter in the begging of summer, but then I forgot. For example index files would be mandatory to BAM files. 

This PR adds a functionality.

**How to test**:
1. how to install it

`git co fix-load_cmd-190820`

1. how to test it, possibly with real cases/data

a . Edit `config.yaml` and comment out `sample_id`
b. Try to load the configuration (I run in demo mode)
`scout --demo load case <path.to.config.yaml> -u`
c. Restore `config.yaml`, load the configuration -it should now load

**Expected outcome**:
KeyError is caught and is printed as Error. Stack trace is printed as debug. It might look like this:

`2019-08-20 12:55:07 mikaellmbp scout.commands.load.case[60571] ERROR KEYERROR 'sample_id' missing when loading 'scout/demo/643594.config.yaml'`

**Review:**
